pom.xml

```diff
@@ -268,6 +269,12 @@
                 <artifactId>kafka-oauth-keycloak-authorizer</artifactId>
                 <version>${strimzi-oauth.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.strimzi</groupId>
+                <artifactId>systemtest-doc</artifactId>
+                <version>${project.version}</version>
+                <scope>compile</scope>
+            </dependency>
```

Comment:
Is this really needed if it is part of this repo?

Reply:
Will check it, thanks :)